### PR TITLE
Remove Prometheus from Metrics guide - no longer relevant, is now a separate monitoring operator guide

### PR DESCRIPTION
**Pull request by david-igou**
With the introduction of the Monitoring Operator this should no longer be in the metrics guide for 3.10 and 3.11.

I cannot attach labels, but this should be enterprise-3.10 and enterprise-3.11 - Not sure about origin.

@openshift/team-documentation

### CODE REVIEW

File: install_config/cluster_metrics.adoc [hunk 1]
```diff
@@ -39,7 +39,7 @@ Hawkular Metrics.
 
 The storage volume metrics available on the kubelet are not available through
 the `/stats` endpoint, but are available through the `/metrics` endpoint. See
-{product-title} via Prometheus for detailed information.
+xref:../install_config/prometheus_cluster_monitoring.adoc[Prometheus Monitoring] for detailed information.
```
Comment (ahardin-rh) [inline install_config/cluster_metrics.adoc]: xref:../install_config/prometheus_cluster_monitoring.adoc#prometheus-cluster-monitoring[Prometheus Monitoring]

This _should_ fix the build failure.

**Comment by ahardin-rh** [comment]
@David-Igou Thank you! 
@anpingli Can you please provide a QE review to verify these changes?

**Comment by ahardin-rh** [comment]
@David-Igou Thank you! Once we get QE verification, I will merge.

**Comment by ahardin-rh** [comment]
@juzhao Can you please provide a QE review?

**Comment by juzhao** [comment]
@ahardin-rh
LGTM, just one reminder: no need to merge to 3.10, cluster monitoring is officially supported since 3.11

**Pull request merged**